### PR TITLE
feat: add missing impact display when no report

### PR DESCRIPTION
**Pull request by benfurber**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

PP don't have reports for each impact year, so this adds a simpler display for those years.

## Screenshots/Videos

Screenshot 2024-01-15 at 12 11 35

### CODE REVIEW

File: src/pages/User/impact/constants.ts [hunk 1]
```diff
@@ -1,10 +1,7 @@
 import type { IImpactYear } from 'src/models'
 
 export const IMPACT_REPORT_LINKS = {
-  2023: 'https://www.preciousplastic.com/impact/2023',
-  2022: 'https://www.preciousplastic.com/impact',
-  2021: 'https://www.preciousplastic.com/impact',
-  2020: 'https://www.preciousplastic.com/impact',
+  2022: 'https://www.preciousplastic.com/impact/2023',
```
Comment (evakill) [inline src/pages/User/impact/constants.ts:4]: should be 2023?

File: src/pages/User/impact/labels.ts [hunk 2]
```diff
@@ -14,12 +14,14 @@ export const invisible = {
...
     link: 'Impact Report',
   },
   owner: {
     label: "Looks like you haven't added your Impact Data yet.",
     link: 'Enter impact data',
   },
 }
+
+export const reportYearLabel =
+  'Until they do you can have a pick of the global impact report! :)'
```
Comment (evakill) [inline src/pages/User/impact/labels.ts:27]: "have a pick of" feels very British 😂

how about "in the meantime, check out the global impact report! :)"

File: src/pages/User/impact/constants.ts (hunk 1 above)
Comment (benfurber) [reply src/pages/User/impact/constants.ts:4]: No. Annoyingly it is 2023 on the parent PP website.

File: src/pages/User/impact/labels.ts (hunk 2 above)
Comment (benfurber) [reply src/pages/User/impact/labels.ts:27]: Haha. Did I write this copy...?

Comment (benfurber) [reply src/pages/User/impact/labels.ts]: Resolved in `c4949c3`

**Review by thisislawatts — APPROVED** [review]

**Pull request merged**